dbt_platform_helper/utils/files.py

```diff
@@ -7,6 +9,8 @@
 from jinja2 import Environment
 from jinja2 import FileSystemLoader
 
+platform_helper_config_file = ".platform-helper-config.yml"
```

Comment:
Is this referring to what we would normally call the `platform_config_file`?

Reply:
No - its the cache file.

Reply:
Can we rename it to make that more obvious.